stl/inc/format

```diff
@@ -2946,13 +2957,13 @@ using format_args  = basic_format_args<format_context>;
 using wformat_args = basic_format_args<wformat_context>;
 
 template <class _Context = format_context, class... _Args>
-_NODISCARD auto make_format_args(const _Args&... _Vals) {
-    return _Format_arg_store<_Context, _Args...>{_Vals...};
+_NODISCARD auto make_format_args(_Args&&... _Vals) {
+    return _Format_arg_store<_Context, _Args...>{_STD forward<_Args>(_Vals)...};

```

Comment:
I'm forwarding because libfmt forwards here, I think you're right though and they forward everywhere but then specifically undo what the forwarding would have done when `make_arg` binds them all to the fmt equivalent to `const _Storage_type<remove_cvref_t<T>>&`

>Have you submitted an LWG issue to fix the preconditions on these functions to strip cvref-quals from the Ti?

no, but Tim Song did.

Reply:
wait is this a separate issue from LWG-3631?

Reply:
Yes it is - [format.arg.store]/2. We can do this as a followup - it doesn't need to block merging.